uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -318,30 +284,34 @@ private fun AppBar(
 private fun FeedList(
     feedContents: FeedContents,
     fmPlayerState: FmPlayerViewModel.State?,
-    isHome: Boolean,
+    feedTab: FeedTab,
     onClickFeed: (FeedItem) -> Unit,
     onFavoriteChange: (FeedItem) -> Unit,
     onClickPlayPodcastButton: (FeedItem.Podcast) -> Unit,
     listState: LazyListState,
-    onDragStopped: (Float) -> Unit,
-    draggableState: DraggableState,
-    robotAnimValue: Float,
-    isListFinished: MutableState<Boolean>,
 ) {
+    val isHome = feedTab is FeedTab.Home
+    val isListFinished by remember {
```

Comment:
You don't have to do this PR, but do you think it's better to separate FeedList into Stateless and Statefull? 👀